spec/controllers/users/verify_profile_controller_spec.rb

```diff
@@ -1,28 +1,67 @@
 require 'rails_helper'
 
-RSpec.describe Users::VerifyProfileController do
+describe Users::VerifyProfileController do
   include Features::LocalizationHelper
 
+  let(:has_pending_profile) { true }
+  let(:success) { true }
+  let(:otp) { 'abc123' }
+  let(:submitted_otp) { otp }
+  let(:pii_attributes) { Pii::Attributes.new_from_hash(otp: otp) }
+  let(:pending_profile) { build(:profile) }
+
+  def stub_decorated_user(user)
```

Comment:
cool - perhaps an opp to create a helper method with all that stubbing and re-use it for each file?